Assets/Scripts/InventoryManager.cs

```diff
@@ -4,17 +4,90 @@
 
 public class InventoryManager : MonoBehaviour
```

Comment:
What's the intent of this class? Should all these fields be replaced with a list?